release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -141,6 +141,13 @@ A xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.
 When configuring an egress firewall rule, you can now use a xref:../networking/ovn_kubernetes_network_provider/configuring-egress-firewall-ovn.adoc#domain-name-server-resolution_configuring-egress-firewall-ovn[DNS domain name] instead of an IP address.
 With the addition of DNS support in the OVN-Kubernetes cluster network provider egress firewall implementation, parity is achieved with the OpenShift SDN cluster network provider egress firewall implementation.
 
+[id="ocp-4-7-preparation-HAProxy-2.2"]
+==== Adjusting HTTP header names
+
+If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
+
+{product-title} is going to update to HAProxy 2.2, which down-cases HTTP header names by default, for example, changing `Host: xyz.com` to `host: xyz.com`. Make sure to add the necessary configuration using `spec.httpHeaders.headerNameCaseAdjustments` before upgrading {product-title} when HAproxy 2.2 is available.
+
```

Comment:
A note on the vague warning: we have guidance in place to avoid assigning specific release versions to upcoming features. In this RN I am looking to make it clear that this change is coming and action must be taken before upgrading.